### ISSUE TITLE
Could not write the "birth" message down the event pipe on arm v5

### ISSUE THREAD

**Issue by bcbcarl** [Snaipe/Criterion#341]
criterion: 2.3.3
glibc: 2.22
arch: armv5te

```
[ERR ] Could not write the "birth" message down the event pipe: Success.
[ERR ] Could not write the "death" message down the event pipe: Success.
Aborted
```

**Comment by Snaipe** [comment]
That is indeed puzzling -- and the fact that errno is 0 is certainly not helping. Is this reproducible on qemu?

**Comment by MrAnno** [comment]
I've been testing Criterion on various real ARM hardware (ARMv6Z, ARMv7-A, ARMv8-A), but I haven't been able to reproduce such an error so far. All ARM-related issues seem to be fixed by #399.

(Unfortunately, I don't have an ARMv5TE machine.)

**Comment by Snaipe** [comment]
I'll assume this will be fixed by v2.4.0. Feel free to reopen or make a new issue if this still happens.
